pkgs/development/python-modules/python-idzip/default.nix

```diff
@@ -0,0 +1,69 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, fetchpatch
+
+, pythonOlder
+
+, setuptools
+
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "python-idzip";
+  version = "0.3.9";
+  pyproject = true;
+
+  disabled = pythonOlder "3.7";
+
+  src = fetchFromGitHub {
+    owner = "bauman";
+    repo = "python-idzip";
+    rev = "refs/tags/${version}";
+    hash = "sha256-ChzwC/Afn0qeo5anq4anIu2eI9i6XDnSvB7jAwY7rSw=";
+  };
+
+  patches = [
+    # fix collision
+    (fetchpatch {
+      name = "fix-bin-folder-collisions.patch";
+      url = "https://github.com/ViZiD/python-idzip/commit/cdb405d3ecca5845a10ce7503f477a4ee6d8db65.patch";
```

Comment:
The patch should come from https://github.com/bauman/python-idzip not a fork. It also helps if the URL to the PR is added to keep track if the patch was merged.